spec/factories/fitbit_profiles.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :fitbit_profile do
+    after(:create) do |fp, evaluator|
```

Comment:
Unused block argument - `evaluator`. If it's necessary, use `_` or `_evaluator` as an argument name to indicate that it won't be used.